.changeset/thin-poems-watch.md

```diff
@@ -0,0 +1,9 @@
+---
+'@backstage/plugin-auth-backend': patch
+---
+
+---
+
+## '@backstage/plugin-auth-backend': patch
```

Comment:
This needs a bit of cleanup

Reply:
This isn't quite formatted properly still. Could look at one of the other changesets or create a new one using the CLI

Reply:
expected format is

```
---
'@backstage/plugin-auth-backend': patch
---
```

Reply:
Pushed a fix so we can get this in today's release